scripts/looker.py

```diff
@@ -32,32 +33,30 @@ def _get_looker_ping_explore(
 ):
     ping_name_snakecase = stringcase.snakecase(ping_name)
     if _looker_explore_exists(looker_namespaces, app_name, ping_name_snakecase):
-        url = f"https://mozilla.cloud.looker.com/explore/{app_name}/{ping_name_snakecase}"
+        url = furl("https://mozilla.cloud.looker.com/explore/{app_name}/{ping_name_snakecase}")
         # if there are multiple channels, we need a channel identifier
         if len(app_group["app_ids"]) > 1 and app_channel:
-            channel_identifier = "mozdata." + table_name.replace("_", "%5E_")
-            url += f"?f[{ping_name_snakecase}.channel]={channel_identifier}"
```

Comment:
This was the source of the problem: later in the code we had the assumption that this *was* the case, and we were just appending further url parameters onto the end with `&`. Using furl fixes that assumption.